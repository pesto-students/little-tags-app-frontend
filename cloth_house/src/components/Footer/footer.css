.footer {
  z-index: 1;
  width: 100%;
  background-color: #006892;
  padding: 10px;
  color: white;
  /* position: fixed; */
  left: 0;
  bottom: 0;
}

.footer button {
  color: white;
}
.positionFixed {
  position: fixed;
}
