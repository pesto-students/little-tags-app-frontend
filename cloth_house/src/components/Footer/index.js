import { React,useContext } from "react";
import { Input, Row, Col, Select, Space, Menu, Dropdown, Button, Carousel ,Card  } from "antd";
import { DownOutlined } from "@ant-design/icons";
import LangContext,{langOptions} from "../../context/LangContext.js"
import { injectIntl } from "react-intl";
import "./footer.css";
import { Footer } from "antd/lib/layout/layout";



const SmileBagFooter = (props) => {
    return (<Footer className="footer">
    <Space>
    <a href="#">{props.intl.formatMessage({id:"app.components.LangSwitch.footer.terms"})}</a> |
    <a href="#">{props.intl.formatMessage({id:"app.components.LangSwitch.footer.madeInIndia"})}</a>
    </Space>
    </Footer>);
}


export default injectIntl(SmileBagFooter)