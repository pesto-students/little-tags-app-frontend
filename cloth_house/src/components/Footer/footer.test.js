import createComponentWithIntl from '../../utils/helper_snapshot'
import Footer from "./index"

it('Footer Snapshot', () => {
    const component = createComponentWithIntl(<Footer />);
    let tree = component.toJSON()
  
    expect(tree).toMatchSnapshot()
  
  });