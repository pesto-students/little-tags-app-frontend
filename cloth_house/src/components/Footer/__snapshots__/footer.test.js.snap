// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot 1`] = `
<footer
  className="ant-layout-footer footer"
>
  <div
    className="ant-space ant-space-horizontal ant-space-align-center"
    style={Object {}}
  >
    <div
      className="ant-space-item"
      style={
        Object {
          "marginRight": 8,
        }
      }
    >
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      />
    </div>
    <div
      className="ant-space-item"
      style={
        Object {
          "marginRight": 8,
        }
      }
    >
       |
    </div>
    <div
      className="ant-space-item"
      style={Object {}}
    >
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      />
    </div>
  </div>
</footer>
`;

exports[`Footer Snapshot 1`] = `
<footer
  className="ant-layout-footer footer"
>
  <div
    className="ant-space ant-space-horizontal ant-space-align-center"
    style={Object {}}
  >
    <div
      className="ant-space-item"
      style={
        Object {
          "marginRight": 8,
        }
      }
    >
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      />
    </div>
    <div
      className="ant-space-item"
      style={
        Object {
          "marginRight": 8,
        }
      }
    >
       |
    </div>
    <div
      className="ant-space-item"
      style={Object {}}
    >
      <button
        className="ant-btn ant-btn-link"
        onClick={[Function]}
        type="button"
      />
    </div>
  </div>
</footer>
`;
